base/sparse/sparsematrix.jl

```diff
@@ -44,6 +44,16 @@ modifications to the returned vector will mutate `A` as well. See `rowvals(A)`
 and `nzrange(A, col)`.
 """
 nonzeros(S::SparseMatrixCSC) = S.nzval
+
+"""
+    rowvals(A)
+
+Return a vector of the row indices of `A`, and any modifications to the returned
+vector will mutate `A` as well. Given the internal storage format of sparse
+matrices, providing access to how the row indices are stored internally can be
+useful in conjuction with iterating over structural nonzero values. See
```

Comment:
This is a long sentence and also kind of awkward. Not sure how to improve? I might remove everything before the first comma in the "Given..." sentence.